panda/src/app/layout.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./globals.css";
+import Header from "./../components/layout/Header";
+import Footer from "../components/layout/Footer";
+
+function RootLayout({ children }) {
+  return (
+    <html lang="ko">
+      <body>
+        <Header />
+        {children}
+        <Footer />
```

Comment:
아래처럼 main 으로 감싸주면 더 HTML 규격에 잘 맞을 것 같네요!
```suggestion
        <Header />
        <main>
          {children}
        </main>
        <Footer />
```